docs/docs/templates/introduction.md

```diff
@@ -238,6 +238,17 @@ Marten.templates.get_template("foo/bar.html")
 
 This will return a compiled [`Template`](pathname:///api/dev/Marten/Template/Template.html) object that you can then render by using a specific context.
 
+### Customizing Template Loaders
+
+The [`templates.loaders`](../development/reference/settings.md#loaders) setting provides fine-grained control over how Marten discovers and loads templates. This setting expects an array of template loader classes, which must inherit from `Marten::Template::Loader::Base`. This setting will overwrite the default template loaders
+configured by Marten. This gives you the option to configure [custom loaders](./how-to/create-custom-loaders.md) to load templates from different sources, such as databases or in-memory data structures.
+
+Example: Configuring a File System Loader
+
+```crystal
+config.templates.loaders = [Marten::Template::Loader::FileSystem.new("/path/to/templates")] of Marten::Template::Loader::Base
+```
```

Comment:
I think it would be better for the overall structure of this introduction page to put this in an "tip" admonition block. For example:

```suggestion
:::tip Customizing template loaders
The [`templates.loaders`](../development/reference/settings.md#loaders) setting provides fine-grained control over how Marten discovers and loads templates. This setting expects an array of template loader classes, which must inherit from `Marten::Template::Loader::Base`. This setting will overwrite the default template loaders
configured by Marten. This gives you the option to configure [custom loaders](./how-to/create-custom-loaders.md) to load templates from different sources, such as databases or in-memory data structures.

For example, the following can be done to configure a file system loader by making use of this setting:

```crystal
config.templates.loaders = [Marten::Template::Loader::FileSystem.new("/path/to/templates")] of Marten::Template::Loader::Base
```
:::
```